frontend/src/components/AddEventForm.js

```diff
@@ -0,0 +1,91 @@
+import {Autocomplete, Button, TextField} from "@mui/material";
+import {DatePicker, LocalizationProvider} from "@mui/lab";
+import DateAdapter from "@mui/lab/AdapterMoment";
+import styled from "styled-components";
+import {useState} from "react";
+import axios from "axios";
+
+export default function AddEventForm() {
+
+    const cities = [
+        {label: 'Hamburg'},
+        {label: 'Berlin'},
+        {label: 'Stuttgart'},
+    ];
+
+    const genres = [
+        {label: 'Concerts'},
+        {label: 'Sport'},
+        {label: 'Comedy'},
+    ];
+
+    const [eventToAdd, setEventToAdd] = useState({
+        name: "",
+        city: "",
+        genre: "",
+    })
+
+    const handleNameChange = (event) => {
```

Comment:
Your Methods a quite clean and easy to understand - that's way it's fine to let it this way. But you could put all you methods together in one method adding a value attribute. Christopher used this feature in the Login Form of the Todo app ` const handleChange = event => {
        setCredentials({...credentials, [event.target.name]: event.target.value})
    }`